### PR TITLE
1781 Document LAMMPS on Stanage

### PR DESCRIPTION
**Pull request by Carldkennedy**
Both toolchains passed testing

Also updating ShARC LAMMPS docs (code blocks were indented and links updated)

### CODE REVIEW

File: stanage/software/apps/lammps.rst [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+.. code-block:: bash
+
+   #!/bin/bash
+   #SBATCH --job-name=mpi-lmp-indent
+   #SBATCH --mail-user=a.person@sheffield.ac.uk
+   #SBATCH --mail-type=FAIL
+   #SBATCH --time=00:01:00
+   #SBATCH --mem=1G
+   #SBATCH --nodes=1
```
Comment (jkwmoore) [inline stanage/software/apps/lammps.rst:108]:
```suggestion
```

Comment (Carldkennedy) [reply stanage/software/apps/lammps.rst]: Added note. Thanks @jkwmoore

**Review by nikishe — COMMENTED** [review]
LGTM.

File: stanage/software/apps/lammps.rst [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+
+Serial (one core) Batch usage
+-----------------------------
+
+First we will copy an example to run:
+
+.. code-block:: bash
+
+   cp /opt/apps/testapps/el7/software/staging/LAMMPS/3Mar2020-intel-2020a-Python-3.8.2-kokkos/examples/indent/in.indent .
```
Comment (jkwmoore) [inline stanage/software/apps/lammps.rst:41]: Can this be amended to use the ``$EBROOTLAMMPS`` path, to so this won't need further amendment when we port the area and / or staging --> live.

Comment (jkwmoore) [reply stanage/software/apps/lammps.rst:41]: ```EBROOTLAMMPS=/opt/apps/testapps/el7/software/staging/LAMMPS/3Mar2020-foss-2020a-Python-3.8.2-kokkos```

**Review by jkwmoore — CHANGES_REQUESTED** [review]
LGTM pending an amendment of the path I commented on. For future PRs, if we can keep corrections existing modules in another PR that'd be grand (makes for a more coherent commit log.)

File: stanage/software/apps/lammps.rst (hunk 2 above)
Comment (Carldkennedy) [reply stanage/software/apps/lammps.rst:41]: Perfect. Thanks @jkwmoore

**Review by jkwmoore — APPROVED** [review]

**Review by nikishe — APPROVED** [review]

**Pull request merged**